src/lib/Scenes/Fair/Screens/__tests__/FairExhibitors-tests.tsx

```diff
@@ -29,9 +29,27 @@ it("renders properly", async () => {
         }
       `}
       mockResolvers={{
-        Fair: () => FairFixture,
+        Fair: () => fairFixture,
       }}
     />
   )
-  expect(tree.html()).toMatchSnapshot()
+  const htmlDom = tree.text()
+
+  // Title
+  expect(htmlDom).toContain("Exhibitors")
+
+  // A "random" set of the exhibitors, we should show the letter
+  // and all the exhibitors
+  const exhibitors = fairFixture.exhibitors_grouped_by_name[1]
+  expect(htmlDom).toContain(exhibitors.letter)
+
+  // The fixtured data is too big for me to hand-edit all of the exhibitors
+  // data into the shape we ask for
+  const anyExhibitors = exhibitors.exhibitors as any[]
+
+  // Because this uses flatlist you can't ask for all of the
+  // exhibitors in the array
+  expect(htmlDom).toContain(anyExhibitors[0].name)
+  expect(htmlDom).toContain(anyExhibitors[1].name)
+  expect(htmlDom).toContain(anyExhibitors[2].name)
 })
```

Comment:
This last test wasn't too valuable, as it'd for people to read a massive machine-readable string, so now it checks that the text we expect is on the screen instead

Reply:
Yup, makes sense